lib/pronto/github.rb

```diff
@@ -91,7 +95,7 @@
 
     def client
       @client ||= Octokit::Client.new(api_endpoint: @config.github_api_endpoint,
-                                      web_endpoint: @config.github_web_endpoint,
+                                    web_endpoint: @config.github_web_endpoint,
```

Comment:
Layout/HashAlignment: Align the keys of a hash literal if they span more than one line.